lib/config_builder/model/provisioner/shell.rb

```diff
@@ -13,12 +13,23 @@ class ConfigBuilder::Model::Provisioner::Shell < ConfigBuilder::Model::Base
   #   @return [String] A string acting as an argument vector to the command.
   def_model_attribute :args
 
+  # @!attribute [rw] run
+  #   @return [String] Defaults to not set. If set to 'always' will cause provisioner to always run.
+  def_model_attribute :run
+
+  def initialize
+    @defaults = {
+     :run => 'once',
```

Comment:
Because I forgot to fix them - will get another commit in to update the other provisioners this afternoon.